plasmapy/physics/distribution.py

```diff
@@ -58,35 +63,460 @@ def Maxwellian_1D(v: units.m/units.s,
+        
+    vz: Quantity
+        The velocity in z-direction units convertible to m/s
+
+    T: Quantity
+        The temperature, preferably in Kelvin
+
+    particle: string, optional
+        Representation of the particle species(e.g., 'p' for protons, 'D+'
+        for deuterium, or 'He-4 +1' for $He_4^{+1}$ : singly ionized helium-4),
+        which defaults to electrons.
+        
+    Vx_drift: Quantity, optional
+        The drift velocity in x-direction units convertible to m/s
+        
+    Vy_drift: Quantity, optional
+        The drift velocity in y-direction units convertible to m/s
+        
+    Vz_drift: Quantity, optional
+        The drift velocity in z-direction units convertible to m/s
+    
+    vTh: Quantity, optional
+        Thermal velocity (most probable) in m/s. This is used for
+        optimization purposes to avoid re-calculating vTh, for example
+        when integrating over velocity-space.
+        
+    units: string, optional
+        Selects whether to run function with units and unit checks (when
+        equal to "units") or to run as unitless (when equal to "unitless").
+        The unitless version is substantially faster for intensive
+        computations.
+
+    Returns
+    -------
+    f : Quantity
+        probability in Velocity^-1, normalized so that:
+        $\iiint_{0}^{\infty} f(\vec{v}) d\vec{v} = 1}
+
+    Raises
+    ------
+    TypeError
+        The parameter arguments are not Quantities and
+        cannot be converted into Quantities.
+
+    UnitConversionError
+        If the parameters is not in appropriate units.
+
+    ValueError
+        If the temperature is negative, or the particle mass or charge state
+        cannot be found.
+
+    Notes
+    -----
+    In one dimension, the Maxwellian speed distribution function describing
+    the distribution of particles with speed v in a plasma with temperature T
+    is given by:
+
+    .. math::
+    f = (\pi * v_Th^2)^{-3/2} \exp(-(\vec{v} - \vec{V_{drift}})^2 / v_Th^2)
+    where v_Th = \sqrt(2 k_B T / m) is the thermal speed
+
+    See also
+    --------
+    Maxwellian_1D
+
+    Example
+    -------
+    >>> from plasmapy.physics import Maxwellian_velocity_3D
+    >>> from astropy import units as u
+    >>> v=1*u.m/u.s
+    >>> Maxwellian_velocity_3D(vx=v,
+    ... vy=v,
+    ... vz=v,
+    ... T=30000*u.K,
+    ... particle='e',
+    ... Vx_drift=0*u.m/u.s,
+    ... Vy_drift=0*u.m/u.s,
+    ... Vz_drift=0*u.m/u.s)
+    <Quantity 3.985430307328085e-20 s3 / m3>
+    
+    
+    """
+    if units == "units":
+        # unit checks and conversions
+        # checking velocity units
+        vx = vx.to(u.m/u.s)
+        vy = vy.to(u.m/u.s)
+        vz = vz.to(u.m/u.s)
+        # catching case where drift velocities have default values, they
+        # need to be assigned units
+        if Vx_drift == 0:
+            if not isinstance(Vx_drift, astropy.units.quantity.Quantity):
+                Vx_drift = Vx_drift * u.m/u.s
```

Comment:
The three `Vdirection_drift = ... ` lines don't get covered by test. Adding another small one with a floating point value would solve that!

Reply:
This refers to tests in `test_distribution.py`, correct?

Reply:
Yeah, that it does.